demo-ng/app/item/item-detail.component.ts

```diff
@@ -1,28 +1,63 @@
-import {Component, OnInit} from "@angular/core";
-import {ActivatedRoute} from "@angular/router";
+import { ScanOptions } from './../../platforms/android/app/src/main/assets/app/tns_modules/nativescript-barcodescanner/barcodescanner-common.d';
```

Comment:
I'm afraid your IDE resolved the wrong import, can you fix it perhaps?

`import { BarcodeScanner, ScanOptions } from "nativescript-barcodescanner";`

Reply:
Oops. That's pushing on Friday evenings is a "no-no"... even for minor sample stuff. 
Cheers for catching it, @EddyVerbruggen and great plugin!